pkgs/by-name/qr/qrtool/package.nix

```diff
@@ -6,18 +6,18 @@
 , installShellFiles
 }:
 
-rustPlatform.buildRustPackage rec {
+rustPlatform.buildRustPackage {
   pname = "qrtool";
-  version = "0.8.5";
+  version = "0.9.0";
 
   src = fetchFromGitHub {
     owner = "sorairolake";
     repo = "qrtool";
-    rev = "v${version}";
-    sha256 = "sha256-jrvNZGO1VIDo6Mz3NKda1C7qZUtF9T00CAFK8yoGWjc=";
+    rev = "v0.9.0";
```

Comment:
I'm not sure why you made this change but it seems fine

Reply:
I am not sure why either, I would revert

Reply:
sorry I've now read the reason for removing `rec` may I suggest still adding `let version = "0.9.0";  in rustPlatform` 

until everything is migrated to `finalAtts`

Reply:
> sorry I've now read the reason for removing rec may I suggest still adding let version = "0.9.0";  in rustPlatform

I did this.

Reply:
I undid this.